package.json

```diff
@@ -55,6 +55,7 @@
     "puppeteer": "^2.1.1",
     "superstruct": "^0.8.3",
     "typescript": "^3.6.4",
-    "why-is-node-running": "^2.1.2"
+    "why-is-node-running": "^2.1.2",
```

Comment:
Can we remove this pkg. It is not been used anywhere.

Reply:
It is `require`d here: https://github.com/Holo-Host/holo-envoy/blob/26dee9a29fab6d761610126452fd1460a2b13c79/tests/unit/test_server_mock_conductor.js#L8

I think it's nice to keep this package around since I've been confused at times why node is running and I often have forgotten to close some websocket and this would help debug that